source/wp-content/themes/wporg-news-2021/sass/components/bottom-banner/_bottom-banner.scss

```diff
@@ -10,15 +10,25 @@
 	margin-top: 0;
 	gap: 0;
 	background-color: var(--wp--preset--color--white);
-	line-height: 30px;
 
 	h3 {
-		font-size: 26px;
 		margin-bottom: var(--wp--custom--margin--baseline);
+		font-size: clamp(26px, 5.4vw, 36px);
+		line-height: 1.625em; /* stylelint-disable-line declaration-property-unit-allowed-list -- This has to be dynamic and relative to the scaled font size. */
```

Comment:
Leaving it unitless will also scale it to the font size. Functionally it's the same, a line height of `1.625em` = 1.625 * this element's font size; `1.625` = 1.625 * current element font size — the only difference is if there's a child element with a different font-size, [the inherited line-height is fixed to the parent font size](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values). Unlikely here, but that unexpected behavior is why unitless is preferred.

Reply:
Ah, I didn't realize that, thanks! Fixed 👍🏻